models/pathology_nuclick_annotation/configs/metadata.json

```diff
@@ -14,8 +15,9 @@
         "nibabel": "4.0.1",
         "pytorch-ignite": "0.4.9"
     },
-    "task": "Pathology Nuclick segmentation",
-    "description": "A pre-trained model for Nuclei Classification within Haematoxylin & Eosin stained histology images",
+    "name": "Pathology nuclick annotation",
+    "task": "Pathology Nuclick annotation",
+    "description": "A pre-trained model for segmenting nuclei cells with user clicks/interactions",
```

Comment:
Hi @SachidanandAlle , here I did a few changes of the description according to the readme file, and it seems the old version is from the classification bundle.
Could you please help to double confirm these three nuclei bundles? The "name" tag added here is to define the display name in the webpage (https://monai.io/model-zoo.html). Thanks in advance!

Reply:
LGTM